reading-json/reading_json.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""reading_json.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1wQxQ6I7oKHQA8qT-OkVuahZDJGboFnd-
+"""
+
+import json
+
+with open("json_file.txt","r") as f:
+  json_text_list = json.load(f)
+
+for element in json_text_list:
+  for keys, values in element.items():
+    print(str(keys)+ ":"+ str(values))
+  print("===================")
+
+with open("daily_covid_cases.json","r") as f:
+  json_covid_dictionary = json.load(f)
+
+for keys in json_covid_dictionary.keys():
+  print(str(keys))
+  for keys1,values in json_covid_dictionary[keys].items():
+    print(f'{keys1} : {values}')
+  print("\t")
```

Comment:
Should use print("\n") to print a new line (instead of tab). This will actually print two new lines since print() adds a new line at the end by default.